public/app/features/explore/Logs/LogsMetaRow.tsx

```diff
@@ -68,9 +84,22 @@ export const LogsMetaRow = React.memo(
             type: 'application/json;charset=utf-8',
           });
 
-          const fileName = `Explore-logs-${dateTimeFormat(new Date())}.json`;
           saveAs(blob, fileName);
           break;
+        case DownloadFormat.CSV:
+          const dataFrame = logRows[0].dataFrame;
+          const transforms: Array<DataTransformerConfig | CustomTransformOperator> = getLogsExtractFields(dataFrame);
+          transforms.push({
+            id: 'organize',
+            options: {
+              excludeByName: {
+                ['labels']: true,
+                ['labelTypes']: true,
+              },
+            },
+          });
+          const transformedDataFrame = await lastValueFrom(transformDataFrame(transforms, [dataFrame]));
+          downloadDataFrameAsCsv(transformedDataFrame[0], fileName);
```

Comment:
Did not test, but isn't this generating a weird filename?
https://github.com/grafana/grafana/blob/main/public/app/features/inspector/utils/download.ts#L66

Reply:
`Explore-logs-2024-02-15 07_32_59.json-data-2024-02-15 07_32_59.csv`
yeah lol I'll fix